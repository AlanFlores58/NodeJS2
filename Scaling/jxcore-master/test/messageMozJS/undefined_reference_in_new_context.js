// Copyright & License details are available under JXCORE_LICENSE file





var common = require('../common');
var assert = require('assert');

common.error('before');

var Script = process.binding('evals').NodeScript;

// undefined reference
var script = new Script('foo.bar = 5;');
script.runInNewContext();

common.error('after');