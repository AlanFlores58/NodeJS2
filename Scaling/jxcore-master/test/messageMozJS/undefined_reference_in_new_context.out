before
Error: Uncaught exception at evalmachine.<anonymous> (native:jxcore_js_object *:*)